sdk/cosmosdb/cosmos/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
 ## 3.17.2 (2022-11-15)
-
+- Changes in bulk api to honour size restictions (i.e 2Mb) while creating individual batches.[#23923](https://github.com/Azure/azure-sdk-for-js/issues/23923)
```

Comment:
This should be added under unreleased 3.17.3

Reply:
Ack. Will remove this.